docs/UserGuide.md

```diff
@@ -176,18 +179,18 @@ Examples:
 
 ### Clearing all entries : `clear`
 
-Clears all entries from the address book.
+Clears all entries from the application.
 
 Format: `clear`
-* Will clear:
-  * Contacts added by users
+* Deletes all members, including:
+  * Members added by users
   * Seed data added from `seedData` command
-* User will be prompted to verify the clear command, before carrying out the clearing.
+* User will be prompted to verify the clear command.
 * To bypass the verification prompt, the user can follow `clear` with `--force`. This will clear the address book without additional prompting.
 
 ### Adding points: `addpoints`
 
-Adds points for a member in the loyalty program.
+Add points for a member in the loyalty program.
```

Comment:
I think "Adds" is grammatically correct here:

the 'addpoints' command **adds** points for a member.....